presto-client/src/main/java/com/facebook/presto/client/ClientSession.java

```diff
@@ -51,6 +51,7 @@
     private final Map<String, String> extraCredentials;
     private final String transactionId;
     private final Duration clientRequestTimeout;
+    private final boolean compressionDisabled;
```

Comment:
Can we instead use `compressionEnabled`? Disabled would incur one more flip in mind :)

Reply:
The phrasing is a little weird, but the mechanism here doesn't actually let you force compression- only disable compression. Same thing with the client side logic. Default negotiation happens inside of jetty around the client `Accept-Encoding` header and server's chosen mime-type and `Content-Encoding` headers.

Reply:
I see. But we could probably just flip the boolean when it is used? I'm just worried this might be error-prone.

Reply:
In general I’m worried about the impression that “enabled” will give. To me, that sounds like the client has ultimate control of the decision when in fact, they do not. The client only as the option to opt-out of compression.